pdf_statement_reader/config/za/absa/cheque.json

```diff
@@ -1,14 +1,23 @@
 {
-    "layout": {
```

Comment:
Including comments in the file like this is technically invalid json. Some json parsers will allow it but this is why I marked it up in the README but not the actual config file

Reply:
The right way to do this is actually with json-schema https://json-schema.org/